packages/web3-types/src/web3_base_provider.ts

```diff
@@ -94,12 +94,28 @@ export interface LegacyRequestProvider {
 	): void;
 }
 
-export interface EIP1193Provider<API extends Web3APISpec> {
+export interface SimpleProvider<API extends Web3APISpec> {
```

Comment:
changelog for this in web3-types package is also required,.

Reply:
Done